posix.mak

```diff
@@ -23,21 +37,40 @@ else
 	DOTLIB:=.a
 endif
 
-DFLAGS=$(MODEL_FLAG) -conf= -O -release -dip25 -inline -w -Isrc -Iimport $(PIC)
-UDFLAGS=$(MODEL_FLAG) -conf= -O -release -dip25 -w -Isrc -Iimport $(PIC)
 DDOCFLAGS=-conf= -c -w -o- -Isrc -Iimport -version=CoreDdoc
 
-CFLAGS=$(MODEL_FLAG) -O $(PIC)
+# Set CFLAGS
+CFLAGS=
+ifneq (,$(filter cc% gcc% clang% icc% egcc%, $(CC)))
```

Comment:
@MartinNowak Please no whitelists! What is this for? This broke building 32-bit Phobos on a 64-bit machine for me because the model flag is no longer passed to the gcc wrapper (required to work around issue 12572). Otherwise at least please use `basename` to allow setting CC to an absolute path.